src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -1224,6 +1224,65 @@ static void Main(string[] args)
             edits.VerifyRudeDiagnostics(active);
         }
 
+        [Fact]
+        public void InstanceConstructorWithInitializerWithLambda_Update1()
+        {
+            string src1 = @"
+class C
+{
+    public C() : this((a, b) => { <AS:0>Console.WriteLine(a + b);</AS:0> }) { }
+}";
```

Comment:
Nope, since we are only doing syntax based checking in VerifyRudeDiagnostics. This test is there to ensure that active statement is matched correctly (AS tag). All we need is syntax.